openedxtozim/mooc.py

```diff
@@ -181,16 +183,16 @@ def annexe(self, c):
                     or "courseware" in path
                 ):
                     continue
-                if "wiki" in path:
+                if "wiki" in path and self.include_wiki:
                     self.wiki, self.wiki_name, path = annexe.wiki(c, self)
-                elif "forum" in path:
+                elif "forum" in path and self.include_forum:
                     path = "forum/"
                     (
                         self.forum_thread,
                         self.forum_category,
                         self.staff_user_forum,
                     ) = annexe.forum(c, self)
-                else:
+                elif not self.include_forum and not self.include_wiki:
```

Comment:
This seems incorrect. What happens if you want wiki and forum (`--wiki --forum`), so you have both `self.include_forum` and `self.include_wiki`?
It seems that this would **only include the wiki** or **only include the forum** but all the rest will be discarded.

ie. the rest is processed only if include_forum and include_wiki are false.

Reply:
Oh. Thanks for pointing this mistake out. The update shall fix this.